packages/medusa/src/types/product-category.ts

```diff
@@ -21,6 +28,11 @@ export type UpdateProductCategoryInput = ProductCategoryInput & {
 }
 
 export class AdminProductCategoriesReqBase {
+  @Transform(({ value }) => (value === "null" ? null : value))
```

Comment:
Todo: could we use IsOptional instead?

Reply:
Or does the null becomes a string when it arrives?

Reply:
I'm transforming null string into actual null. isOptional doesn't give me what I wanted. This one works perfectly.